server/package.json

```diff
@@ -19,6 +19,7 @@
     "concurrently": "^7.3.0",
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
+    "crypto": "^1.0.1",
```

Comment:
By the way, I would normally commit the package-lock.json when updating a dependency, but it looks like maybe that hasn't been done for a while so it would include a lot of irrelevant changes that might pollute this PR. You could do that separate anyway. To be honest it probably doesn't matter all that much anyway for a little demo project, but good practice to get into the habit of anyway.